### PR TITLE
Add Italian language

### PR DESCRIPTION
**Pull request by bovirus**
@artar94 

Add Italian language.
Please check, merge and release a new version to test it.
Thanks.

### CODE REVIEW

File: Localization/Italian/ItalianInstaller.iss [hunk 1]
```diff
@@ -0,0 +1,29 @@
+﻿[CustomMessages]
+; InstallType
+english.InstallType=Seleziona tipo installazione
```
Comment (kjen93) [inline Localization/Italian/ItalianInstaller.iss:3]: @bovirus thank you for localization!
The minor issue here is that strings in this file should start from `italian`, not `english`.

**Review by kjen93 — CHANGES_REQUESTED** [review]

File: Localization/Italian/ItalianBatteryModeLanguage.rc [hunk 2]
```diff
@@ -0,0 +1,158 @@
+STRINGTABLE
+LANGUAGE LANG_ENGLISH, SUBLANG_ENGLISH_US
+{
+0, "Italiano"
+1, "Modalità batteria"
```
Comment (kjen93) [inline Localization/Italian/ItalianBatteryModeLanguage.rc:5]: This string was meant to be the name of the program, so let's leave it untranslated :)

**Comment by artar94** [comment]
@bovirus Did you accidentally and knowingly close this pull request?

**Comment by bovirus** [comment]
@artar94 

The tramnslation files rewuired also some changes also about the header and SUBSET_DEFAULT.
I prefere to close this PR and make a new clean PR.

https://github.com/tarcode-apps/BatteryMode/pull/20

**Pull request closed without merging**